.github/workflows/test.yml

```diff
@@ -12,147 +12,110 @@ jobs:
           git diff --no-index --word-diff -- test/data/xy_rate.json test/data/xy_rate.json.expected
           python -c 'import pandas; pandas.testing.assert_frame_equal(pandas.read_csv("test/data/xy_rate.csv"), pandas.read_csv("test/data/xy_rate.csv.expected"))'
       - name: cli-cv-xy-rate-without-metadata
-        shell: bash -l {0}
+        shell: pixi run bash -e {0}
         run: |
-          svgdigitizer cv test/data/xy_rate_without_metadata.svg --si-units
+          pixi run -e ${{ matrix.environment }} svgdigitizer cv test/data/xy_rate_without_metadata.svg --si-units
           git diff --no-index --word-diff -- test/data/xy_rate_without_metadata.json test/data/xy_rate_without_metadata.json.expected
           python -c 'import pandas; pandas.testing.assert_frame_equal(pandas.read_csv("test/data/xy_rate_without_metadata.csv"), pandas.read_csv("test/data/xy_rate_without_metadata.csv.expected"))'
       - name: cli-cv-xy-rate-reference
-        shell: bash -l {0}
+        shell: pixi run bash -e {0}
         run: |
-          svgdigitizer cv test/data/xy_rate_reference.svg --metadata test/data/xy_rate_reference.yaml --si-units
+          pixi run -e ${{ matrix.environment }} svgdigitizer cv test/data/xy_rate_reference.svg --metadata test/data/xy_rate_reference.yaml --si-units
           git diff --no-index --word-diff -- test/data/xy_rate_reference.json test/data/xy_rate_reference.json.expected
           python -c 'import pandas; pandas.testing.assert_frame_equal(pandas.read_csv("test/data/xy_rate_reference.csv"), pandas.read_csv("test/data/xy_rate_reference.csv.expected"))'
       - name: cli-cv-cv-comment
-        shell: bash -l {0}
+        shell: pixi run bash -e {0}
         run: |
-          svgdigitizer cv test/data/cv_comment.svg --metadata test/data/cv_comment.yaml --si-units
+          pixi run -e ${{ matrix.environment }} svgdigitizer cv test/data/cv_comment.svg --metadata test/data/cv_comment.yaml --si-units
           git diff --no-index --word-diff -- test/data/cv_comment.json test/data/cv_comment.json.expected
           python -c 'import pandas; pandas.testing.assert_frame_equal(pandas.read_csv("test/data/cv_comment.csv"), pandas.read_csv("test/data/cv_comment.csv.expected"))'
       - name: cli-cv-xy-rate-without-metadata-skewed
-        shell: bash -l {0}
+        shell: pixi run bash -e {0}
         run: |
-          svgdigitizer cv test/data/xy_rate_without_metadata_skewed.svg --skewed --si-units
-          git diff --no-index --word-diff -- test/data/xy_rate_without_metadata_skewed.json test/data/xy_rate_without_metadata_skewed.json.expected
-          python -c 'import pandas; pandas.testing.assert_frame_equal(pandas.read_csv("test/data/xy_rate_without_metadata_skewed.csv"), pandas.read_csv("test/data/xy_rate_without_metadata_skewed.csv.expected"))'
-      - name: cli-cv-axes-orientation
-        shell: bash -l {0}
-        run: |
-          svgdigitizer cv test/data/axes_orientation.svg --si-units
```

Comment:
Did you ommit this on purpose?

Reply:
No.

Reply:
Probably a merge that I screwed up. Thanks for checking.

Reply:
I added it again